evm/deploy-upgradable.js

```diff
@@ -52,7 +50,8 @@ async function getImplementationArgs(contractName, config, options) {
                 throw new Error(`${config.name} | Missing AxelarDepositService.refundIssuer in the chain info.`);
             }
 
-            const gateway = config.AxelarGateway?.address;
+            const gateway = contractConfig.gatewayAddress ?? config.AxelarGateway?.address;
+            config.AxelarGateway.address = gateway;
```

Comment:
doesn't need to be made configurable i think. at least we shouldn't overwrite the actual AxelarGateway config (as that is reserved to be modified by the gateway deployment script)